core-concepts/streams/stream-names.md

```diff
@@ -25,21 +25,21 @@ A _category_ stream name does not have an ID. For example, the stream name for t
 
 ## Category Types
 
-Category stream names can be broken down into sub-categories, or _types_.
+Stream names can also have an optional list of _types_.
 
-The `command` type is a very common category type. It's used to indicate a stream that is used to transport _commands_ rather than _events_.
+The `command` type is one of the most common types associated with a stream. It's used to indicate a stream that is used to transport _commands_ rather than _events_.
```

Comment:
Well, it's not "associated with a stream". The stream name is the stream's identity. So it's more than merely an association.